src/content/docs/en/guides/astro-db.mdx

```diff
@@ -499,3 +499,21 @@ export default async function() {
 
 The value returned by `asDrizzleTable('Pets', Pets)` is equivalent to `import { Pets } from 'astro:db'`, but is available even when Astro’s type generation can’t run.
 You can use it in any integration code that needs to query or insert into the database.
+
+## Self-hosted production deployment
+
+If you deploy your site to a self-managed host such as a [virtual private server](https://en.wikipedia.org/wiki/Virtual_private_server), you can choose to use a database file instead of connecting to Astro Studio.
+
+:::caution
+Using a database file is an advanced feature, and care should be taken when deploying to prevent overriding your database. This method will also not work in serverless deployments, as the file system is not persisted in those environments. For a fully managed solution, try [connect to the Astro Studio platform](/en/recipes/studio/) instead.
```

Comment:
```suggestion
Using a database file is an advanced feature, and care should be taken when deploying to prevent overriding your database and losing your production data.

Additionally, this method will not work in serverless deployments, as the file system is not persisted in those environments.

For a fully managed solution, [connect to databases hosted on the Astro Studio platform](/en/recipes/studio/) instead.
```